controllers/data.controller.js

```diff
@@ -178,8 +201,8 @@ exports.updateTierAndScore = async (req, res) => {
         isAnswerCorrect,
         questionsAnsweredNb,
       );
-      if (response.status === 200) {
-        return res.json({ message: "Tier and score updated successfully" });
+      if (response.status.code === 200) {
```

Comment:
so, here if there will be problem in response from handlePlacementQuestions, it will throw error. so dont need to use if